### ISSUE TITLE
Browser truncation results different

### ISSUE THREAD

**Issue by dcousens** [crypto-browserify/pbkdf2#40]
**node 6**
```
mnemonicメートルガバヴァぱばぐゞちぢ十人十色

bW5lbW9uaWPjg6Hjg7zjg4jjg6vjgqvjgpnjg4/jgpnjgqbjgpnjgqHjga/jgprjga/jgpnjgY/jgpnjgp3jgpnjgaHjgaHjgpnljYHkurrljYHoibI=
```

**browser**
```
mnemonicãƒ¡ãƒ¼ãƒˆãƒ«ã‚«ã‚™ãƒã‚™ã‚¦ã‚™ã‚¡ã¯ã‚šã¯ã‚™ãã‚™ã‚ã‚™ã¡ã¡ã‚™åäººåè‰² 

bW5lbW9uaWPDo8aSwqHDo8aSwrzDo8aSy4bDo8aSwqvDo+KAmsKrw6PigJrihKLDo8aSwo/Do+KAmuKEosOj4oCawqbDo+KAmuKEosOj4oCawqHDo8KBwq/Do+KAmsWhw6PCgcKvw6PigJrihKLDo8KBwo/Do+KAmuKEosOj4oCawp3Do+KAmuKEosOjwoHCocOjwoHCocOj4oCa4oSiw6XCjcKBw6TCusK6w6XCjcKBw6jigLDCsg==
```

**Comment by dcousens** [comment]
ping @fanatid @calvinmetcalf @dominictarr 
I don't have time to fix this at the moment.

`npm run bundle-test` and load `test/bundle.html` in browser (watch the console).

**Comment by dcousens** [comment]
ping @Sjors

**Comment by dcousens** [comment]
@feross this appears to be an issue in https://github.com/feross/buffer

**Comment by dcousens** [comment]
Test case:
```
var salt = 'mnemonicメートルガバヴァぱばぐゞちぢ十人十色'
var buffer = new Buffer(salt, 'utf8')

console.log(buffer.toString('base64'))
```

Compare the two on browser and node.

**Comment by dcousens** [comment]
Opened issue at https://github.com/feross/buffer/issues/133

**Comment by dcousens** [comment]
Fixed by #42
